tests/xharness/TestImporter/Templates/Managed/Resources/RegisterType.cs

```diff
@@ -3,30 +3,29 @@
 using System.Collections.Generic;
 using Xunit;
 using Xunit.Abstractions;
-%USING%
+_REPLACE_USING_REPLACE_
 
 namespace Xamarin.iOS.UnitTests {
 	// autogenerated class that uses types from the test assemblies so that
 	// they are aot and not linked away.
-	public static class RegisterType
-	{
-		public static bool IsXUnit = %IS XUNIT%;
-		public static Dictionary <string, Type> TypesToRegister = new Dictionary <string, Type> {
-%KEY VALUES%
+	public static class RegisterType {
+		public static bool IsXUnit = _REPLACE_IS_XUNIT_REPLACE_;
+		public static Dictionary<string, Type> TypesToRegister = new Dictionary<string, Type> {
+_REPLACE_KEY_VALUES_REPLACE_
 		};
 
```

Comment:
This looks a little funky to me. Is this expected with the `%USING%` and the `_REPLACES`'s?

Reply:
@tj-devel709 

I split these changes out into a separate PR (#16878), and removed them from this PR.

Reply:
Looks good!